README.docker.md

```diff
@@ -52,6 +49,8 @@ to delete the volumes (this will reset your instance).
 
 ## Post-install commands
 
-1. run the script `DELA_REPLICAS=... init_dela.sh` to initialize the DELA network with `DELA_REPLICAS set to the same value as in .env`
-2. run `docker exec -it d-voting-backend-1 /bin/bash` to connect to the backend
-3. execute `node -e 'require("./dbUtils").addAdmin("./dvoting-users", <sciper>)'` with your Sciper to add yourself as admin
+1. `./init_dela.sh`
+2. `docker compose exec backend /bin/bash`
+3. `cd src`
+4. `node -e 'require("./dbUtils").addAdmin("<sciper>")'`
+5. exit the container and run `docker compose down && docker compose up -d`
```

Comment:
These changes will be affected by this PR https://github.com/dedis/d-voting/pull/289 if it got merged first

Reply:
I'll rebase the branch